docs/root/intro/arch_overview/ssl.rst

```diff
@@ -85,6 +85,126 @@ standard Debian installations. Common paths for system CA bundles on Linux and B
 See the reference for :ref:`UpstreamTlsContexts <envoy_api_msg_auth.UpstreamTlsContext>` and
 :ref:`DownstreamTlsContexts <envoy_api_msg_auth.DownstreamTlsContext>` for other TLS options.
 
+.. _secret_discovery_service:
```

Comment:
shall we put this in a new file like docs/root/configuration/secret.rst and put a link from here?

Reply:
Done.  move it into a separate file secret.rst under configuration.